src/m365/spo/commands/commandset/commandset-add.spec.ts

```diff
@@ -275,4 +276,59 @@ describe(commands.COMMANDSET_ADD, () => {
     }
     assert(false);
   });
+
+  it('includes description in request body when provided', async () => {
+    let actualRequestBody: any;
+
+    sinon.stub(request, 'post').callsFake(async opts => {
+      if ((opts.url === `https://contoso.sharepoint.com/_api/Web/UserCustomActions`)) {
+        actualRequestBody = opts.data;
+        return commandactionResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        webUrl: validWebUrl,
+        title: validTitle,
+        description: validDescription,
+        clientSideComponentId: validClientSideComponentId,
+        listType: validListType
+      }
+    });
+
+    assert.strictEqual(actualRequestBody.Description, validDescription);
+    assert.strictEqual(actualRequestBody.Title, validTitle);
+    assert.strictEqual(actualRequestBody.ClientSideComponentId, validClientSideComponentId);
+    assert.strictEqual(actualRequestBody.RegistrationId, '100'); // List type
+    assert.strictEqual(actualRequestBody.RegistrationType, 1);
```

Comment:
Why are we using these different asserts here? Let's check it all at once like we did for the applicationcustomizer commands.